style.css

```diff
@@ -30,29 +32,130 @@ section {
 .first {
   color: rgba(255, 255, 255, 0.3);
   background: #50233B;
-  position: relative;
-  display: flex;
-  justify-content: flex-end;
-  align-items: flex-end;
-  overflow: hidden;
+  overflow: visible;
+}
+
+.first::before {
+  content: '';
+  position: absolute;
+  width: 12vh;
+  height: 12vh;
+  border-radius: 5%;
+  background: #50233B;
+  left: 10vh;
+  bottom: -5vh;
+  transform: rotate(45deg);
+  z-index: 1;
 }
 
 .second {
   color: rgba(255, 255, 255, 0.3);
   background: #FF9D00;
-  position: relative;
-  display: flex;
-  justify-content: flex-end;
-  align-items: flex-end;
-  overflow: hidden;
+}
+
+.second__image {
+  height: 40%;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  clip-path: polygon(20% 0%, 0% 20%, 30% 50%, 0% 80%, 20% 100%, 50% 70%, 80% 100%, 100% 80%, 70% 50%, 100% 20%, 80% 0%, 50% 30%);
+  filter: blur(10px);
+  transition: filter 500ms;
+}
+
+.second__image:hover {
+  filter: blur(0px)
 }
 
 .third {
   color: rgba(255, 255, 255, 0.3);
   background: #8EA700;
-  position: relative;
-  display: flex;
-  justify-content: flex-end;
-  align-items: flex-end;
-  overflow: hidden;
+}
+
+.third__button {
+  border: none;
+  border-radius: 10px;
+  width: 15vw;
+  height: 5vw;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  text-decoration: none;
+  font: small-caps 800 30px 'Quicksand';
+  background: #92004e;
+  color: #ffffff;
+  cursor: pointer;
+  text-align: center;
+  transition: background 200ms ease-in,
+              box-shadow 200ms ease-in;
+}
+
+.third__button:hover {
+  background: #50233b;
+  box-shadow: 0px 11px 25px 0px rgba(58, 68, 0, 0.5);
+}
+
+.fourth {
+  color: rgba(255, 255, 255, 0.3);
+  background: repeating-linear-gradient(120deg, #004a84 0%, #004a84 5%, #00467b 5%, #00467b 10%);
```

Comment:
turned out to be a really nice background idea!

Reply:
Thank you, I'm quite pleased I can make a background interesting without any images. Might explore some more.